script.js

```diff
@@ -0,0 +1,151 @@
+// Assignment Code
+var acc = 0;
```

Comment:
Could we not use const or let instead of var here? var would be making these variables global